panther_docking/config/panther_docking_server.yaml

```diff
@@ -16,33 +16,21 @@
     dock_plugins: ["panther_charging_dock"]
     panther_charging_dock:
       plugin: panther_docking::PantherChargingDock
-      base_frame: "<robot_namespace>/base_link"
-      docking_distance_threshold: 0.15
-      docking_yaw_threshold: 0.15
-      staging_x_offset: -0.8
-      staging_yaw_offset: 0.0
+      external_detection_timeout: 0.1
+      docking_distance_threshold: 0.08
+      docking_yaw_threshold: 0.1
+      staging_x_offset: -0.5
+      filter_coef: 0.1
 
-      # TODO: @delihus Try to remove this parameters by using docking station description in the ros_components_description
-      # Transform between april tag frame and dock pose. An april tag Z+ faces always a camera
-      external_detection_timeout: 0.3
-      external_detection_translation_x: 0.6
-      external_detection_translation_y: 0.0
-      external_detection_translation_z: 0.0
-      external_detection_rotation_roll: 0.0
-      external_detection_rotation_pitch: 0.0
-      external_detection_rotation_yaw: 3.14
-
-      enable_charger_service_call_timeout: 1.0
-
-    docks: ["main_dock"]
-    main_dock:
+    docks: ["main"]
+    main:
       type: panther_charging_dock
-      frame: <robot_namespace>/main_wibotic_station_link
+      frame: <robot_namespace>/main_wibotic_receiver_requested_pose_link
```

Comment:
The frame of station should be independent form the robot namespace (e.g. two robot can use the same station)

Reply:
Yeah, your are  right.
I will address it in the previous task. It will be okay?

Reply:
ok